client/src/components/layouts/global.css

```diff
@@ -12,83 +12,24 @@ body {
   background: var(--secondary-background);
 }
 
-/* This is to override Tailwind as it sets the font-weight to `bolder`.
-   TODO: Move this rule to @freecodecamp/ui. */
-b,
-strong {
-  font-weight: bold;
-}
-
-/* TODO: Move this rule to @freecodecamp/ui. */
-label {
-  display: inline-block;
-  max-width: 100%;
-  margin-bottom: 5px;
-  font-weight: bold;
-}
-
 /* TODO: Move this rule to @freecodecamp/ui. (Though some styles might need to be dropped). */
 legend {
-  display: block;
-  width: 100%;
-  padding: 0;
-  margin-bottom: 20px;
-  font-size: 21px;
-  line-height: inherit;
   color: #333;
-  border: 0;
   border-bottom: 1px solid #e5e5e5;
 }
 
 /* TODO: Move this rule to @freecodecamp/ui. */
 blockquote {
-  padding: 10px 20px;
-  margin: 0 0 20px;
-  font-size: 17.5px;
   border-left: 5px solid #eeeeee;
 }
 
-/* TODO: Move this rule to @freecodecamp/ui. */
-blockquote footer,
-blockquote small,
-blockquote .small {
-  display: block;
-  font-size: 80%;
-  line-height: 1.42857143;
-  color: var(--gray-45);
-}
```

Comment:
Sorry, I accidentally included these in the issue thread.

Past me decided that the CSS should stay here / not be moved to the UI library: https://github.com/freeCodeCamp/ui/issues/167#issuecomment-2739100273.

Please help revert the removal, and change the comment to:

```
/* TODO: Move this rule to components/Intro/intro.css. */
```